edflow/data/believers/meta_loaders.py

```diff
@@ -64,6 +75,39 @@ def numpy_loader(path):
     """
 
     def loader():
-        return np.load(path)
+        path_ = os.path.join(root, path)
+        return np.load(path_)
+
+    loader.__doc__ = f"""Loads the numpy array found at {path} relative to
+    {root}."""
 
     return loader
+
+
+def category(index, categories):
```

Comment:
this is returned as a default loader but has a different signature than the others (no root kwarg) - is this on purpose?

Reply:
The signatures of all loaders should be allowed to be unique. Some loaders are meant to be for loading things given a path, others can be used to process certain information, like loading a category given its index or maybe in the future converting keypoints into stickmen.

Reply:
makes sense